rotate.c

```diff
@@ -172,6 +206,14 @@ void rotate_init(struct context *cnt)
         cnt->rotate_data.degrees = cnt->conf.rotate_deg % 360; /* Range: 0..359 */
     }
 
+    if ( strncmp(cnt->conf.flip_axis, "h", 1) == 0 ) {
```

Comment:
started using "vertical" and "horizontal", but ended up preferring the shortcut "v" and "h".
This code was a reminiscence (changed)